packages/nvidia_nat_nemo_customizer/src/nat/plugins/customizer/dpo/trainer_adapter.py

```
@@ -0,0 +1,550 @@
+                for episode_item in trajectory.episode:
+                    if isinstance(episode_item, DPOItem):
+                        formatted_prompt = self._format_prompt(episode_item.prompt)
+                        dpo_record = {
+                            "prompt": formatted_prompt,
+                            "chosen_response": episode_item.chosen_response,
+                            "rejected_response": episode_item.rejected_response,
+                        }
+                        all_items.append(dpo_record)
+
+        if not all_items:
+            raise ValueError("No DPO items found in trajectories")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)